packit_service/worker/jobs.py

```diff
@@ -710,38 +717,60 @@ def get_config_for_handler_kls(
 
         return matching_jobs
 
-    def push_initial_metrics(
+    def push_statuses_metrics(
+        self,
+        statuses_check_feedback: List[datetime],
+    ):
+        """
+        Push the metrics about the time of setting initial statuses for the first and last check.
+
+        Args:
+            statuses_check_feedback: A list of times it takes to set every initial status check.
+        """
+        if statuses_check_feedback:
+            pushgateway = Pushgateway()
+            response_time = elapsed_seconds(
+                begin=self.event.created_at, end=statuses_check_feedback[0]
+            )
+            logger.debug(
+                f"Reporting first initial status check time: {response_time} seconds."
+            )
+            pushgateway.first_initial_status_time.observe(response_time)
+            if response_time > 30:
+                pushgateway.no_status_after_30_s.inc()
```

Comment:
I think we want 15, not 30, don't we?

OTOH, it's just about the Sentry reporting so I don't mind if this stays as it is.

Reply:
Yes, I think this is not related with SLO1 so I have not touched it.

Reply:
to me, it would make sense to align it with the SLO1